swingset/src/main/java/com/nqadmin/swingset/formatting/SSFormattedTextField.java

```diff
@@ -308,7 +347,7 @@ public void customInit() {
 		addFocusListener(this);
 
 		// Setting inputVerifier to validate field before focus is lost
-		// See https://docs.oracle.com/javase/8/docs/api/javax/swing/JFormattedTextField.html
+		// See https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/javax/swing/JFormattedTextField.html
```

Comment:
Not sure referencing jdk-17 is a good idea, since this class is using methods that are deprecated in jdk-9 onwards. Or maybe just a comment to that effect.

Reply:
Good catch that the single argument version of shouldYieldFocus() is depreciated. The Java 17 JavaDoc that I reference for JFormattedTextField() provides sample code using the depreciated method. As you suggest elsewhere, we might be able to just get rid of shouldYieldFocus() if it's a copy of the super class code. I just copied it from the JavaDoc example.

Reply:
I think there were two references in the code to the Oracle docs. I put links to the docs for both 8 and 17 as a compromise.